pkg/sbproxy/reconcile/reconcile_brokers.go

```diff
@@ -38,19 +38,19 @@ func (r *ReconciliationTask) processBrokers() {
 		return
 	}
 
-	// get all the registered proxy brokers from the platform
+	// get all the registered brokers from the platform
 	brokersFromPlatform, err := r.getBrokersFromPlatform()
 	if err != nil {
 		logger.WithError(err).Error("An error occurred while obtaining already registered brokers")
 		return
 	}
 
-	// get all the brokers that are in SM and for which a proxy broker should be present in the platform
 	brokersFromSM, err := r.getBrokersFromSM()
 	if err != nil {
 		logger.WithError(err).Error("An error occurred while obtaining brokers from Service Manager")
 		return
 	}
+	r.stats[smBrokersStats] = brokersFromSM
```

Comment:
stats doesn't mean anything to me. Why is it a map? You store only brokers. A slice with name smBrokers might be better.

Reply:
I couldn't make up a name for this property. It is intended to keep already fetched items from Service Manager during current (meaning it is not the same across jobs) reconciliation job.